dunning/dunning/doctype/dunning/dunning_list.js

```diff
@@ -0,0 +1,10 @@
+frappe.listview_settings['Dunning'] = {
+	get_indicator: function (doc) {
+		if (doc.resolution_status === "Resolved") {
+			return [__("Resolved"), "green", "status,=,Resolved"];
```

Comment:
How does the `resolution_status` change to "Resolved"?

A Dunning for a paid Sales Invoice is still marked as "Unresolved".

Reply:
This will be changed manually by the user.